src/pages/signup-affiliates/index.tsx

```diff
@@ -1,13 +1,245 @@
+                        name="email"
+                        type="text"
+                        border="solid 1px var(--color-grey-7)"
+                        label_color="grey-5"
+                        label_hover_color="grey-5"
+                        background="white"
+                        error={email_error_msg}
+                        value={email}
+                        label={localize('Email')}
+                        placeholder={'Email'}
+                        autoFocus={autofocus}
+                        autoComplete="off"
+                        required
+                    />
+                </InputGroup>
+
+                <CaptchaContainer>
+                    <AgreementLabel
+                        isChecked={is_checked}
+                        handleChangeCheckbox={handleChange}
+                        color="#333333"
+                        link_text={localize('I’m not a robot')}
+                    />
+                    <ImageCaptchaContainer>
+                        <img src={Captcha} alt="Captcha" width="32" height="32" />
+                    </ImageCaptchaContainer>
+                </CaptchaContainer>
+
+                <EmailButton type="submit" secondary="true" id="partner-signup">
+                    {localize('Create partner account')}
+                </EmailButton>
+                <SignupWithContainer>
+                    <Line />
+                    <StyledText color="grey-5" align="center">
+                        {localize('Or sign up with')}
+                    </StyledText>
+                    <Line />
+                </SignupWithContainer>
+                <SocialWrapper>
+                    <GoogleWrapper id="google-signup">
+                        <img src={Google} alt="google" width="40" height="40" />
+                    </GoogleWrapper>
+                    <FacebookWrapper id="facebook-signup">
+                        <img src={Facebook} alt="facebook" width="40" height="40" />
+                    </FacebookWrapper>
+                    <AppleWrapper id="apple-signup">
+                        <img src={Apple} alt="apple" width="40" height="40" />
+                    </AppleWrapper>
+                </SocialWrapper>
+                <LoginContainer>
+                    <LoginText mt="3.75rem">
+                        {localize('Already have a Deriv or Binary.com affiliate account?')}{' '}
+                    </LoginText>
+                    <StyledText id="dm-new-login-button" align="center" size="16px" color="red">
+                        {localize('Log in')}
+                    </StyledText>
+                </LoginContainer>
+            </SignUpWrapper>
         </AffiliateSignupLayout>
     )
 }
+AffiliateSignup.prototypes = {
+    autofocus: PropTypes.bool,
+    clearEmail: PropTypes.func,
+    email: PropTypes.string,
+    email_error_msg: PropTypes.string,
+    handleInputChange: PropTypes.func,
+    handleValidation: PropTypes.func,
+}
```

Comment:
No need for proptypes we are already using typescript

Reply:
okay removed